### PR TITLE
Replace placeholder data

### LINKED ISSUE #13: Replace hardcoded data
### Description
- As a user, I'd like to see data I've provided displayed.

### Acceptance Criteria
- Remove hardcoded data used as a placeholder
- Use actual data the user inputs.

### PR DESCRIPTION
**Pull request by zeeberry**
Fixes #13

### CODE REVIEW

File: pages/dashboard.js [hunk 1]
```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
...
-  const interviewDate = 'June 11 2021 @ 12pm ET';
-  const interviewType = 'Technical Interview';
-  const role = 'Software Engineer';
-  const excitement = '2';
-  const nerves = '2';
-
   useEffect(() => {
-    if (!profile.length) {
-      setProfile(getProfile(data))
+    async function fetchAccount() {
```
Comment (zeeberry) [inline pages/dashboard.js:25]: do we need to define a function here, given you're calling it right after definition?

File: pages/dashboard.js [hunk 2]
```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
...
+          setApplications(account.profile.applications.data);
+        }
+        if (errorMessage) {
+          setErrorMessage(errorMessage);
+        }
+      }
     }
-  }, [data, profile.length]);
+    fetchAccount();
+  }, [user, profile]);
```
Comment (zeeberry) [inline pages/dashboard.js:40]: do we want the hook to run when profile changes?

File: pages/dashboard.js [hunk 3]
```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
...
+
+  return (user === undefined) ?
+    (
+      <>
```
Comment (zeeberry) [inline pages/dashboard.js:44]: we dont need these fragments here given you are returning one component (in this case `<section>`)

File: pages/dashboard.js [hunk 4]
```diff
@@ -13,46 +13,79 @@ const Content = styled.div`
...
+                      )
+                    })}
+                    <hr />
+                  </div>)
+              })}
+            </Content>}
+        </>
+      )
+      :
+      <>
```
Comment (zeeberry) [inline pages/dashboard.js:86]: same as above comment.

**Review by zeeberry — COMMENTED** [review]
thanks for picking this up! left some small comments.

File: pages/dashboard.js (hunk 1 above)
Comment (beebzz) [reply pages/dashboard.js:25]: yep, react forces you to define your useEffect as a standalone async function (then call it right after) if you're fetching data asynchronously. see [here](https://stackoverflow.com/questions/53332321/react-hook-warnings-for-async-function-in-useeffect-useeffect-function-must-ret)

Comment (zeeberry) [reply pages/dashboard.js:25]: oh interesting! good to know. 👍🏼

File: pages/dashboard.js (hunk 2 above)
Comment (beebzz) [reply pages/dashboard.js:40]: nope, good catch

**Review by zeeberry — COMMENTED** [review]
LGTM

**Pull request merged**